test/e2e/cluster/create.go

```diff
@@ -42,10 +43,12 @@ type Create struct {
 	stack  *stack
 }
 
-func NewCreate(aws aws.Config, logger logr.Logger) Create {
+func NewCreate(aws aws.Config, logger logr.Logger, endpoint string) Create {
```

Comment:
can we add a comment to this method saying that if the endpoint is an empty string it will default to the prod endpoint?